src/services/api/routes/communities.ts

```diff
@@ -28,22 +28,48 @@ const createNewCommunitySchema = z.object({
 
 const router: express.Router = express.Router();
 
+async function respondCommunityNotFound(response: express.Response) : Promise<void> {
+	response.status(404);
+	response.send(xmlbuilder.create({
+		result: {
+			has_error: 1,
+			version: 1,
+			code: 404,
+			error_code: 919,
+			message: 'COMMUNITY_NOT_FOUND'
+		}
+	}).end({ pretty: true }));
+}
```

Comment:
This function does not use async/await, so it shouldn't be marked as async/return a promise